packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -221,7 +221,14 @@ class AOTSnapshotter {
       ]);
     }
 
+    if (platform == TargetPlatform.android_arm) {
```

Comment:
Huh?

Reply:
Looks like I forgot to re-push after I killed that block. On my phone though so will re-push in the morning.

Reply:
Updated.